apps/client/components/Profile/posts/UserBadge/index.tsx

```diff
@@ -25,7 +25,7 @@ const UserBadge = () => {
       `}
     >
       <Image
-        src={userInfo?.profile || avatar}
+        src={userInfo?.profile || grayAvatar}
```

Comment:
```suggestion
        src={userInfo?.profile ??> grayAvatar}
```

nullish 병합 연산자를 사용하여 마우스를 굳이 올리지 않아도 앞의 profile 값이 undefined 값을 가질수 있다는 것을 알려주는 것은 어떤가요? || 연산자는 만족하는 수많은 조건이 있어 특정 조건을 가진 문법으로 변경하는 것이 더 좋다고 생각합니다!

혹은 해당 논리 연산자를 사용해야만 했다면 이유를 알려주세요!

Reply:
잘 이해를 못했습니다 현재는
userInfo.profile 이 undefined이면 grayAvatar를 보여줘라라는 의미인데. 
결국 ||나, ?? 가 똑같지 않나요?

userInfo.profile은 값이 있거나, undefined둘중 무조건 하나니까요.

> 마우스를 굳이 올리지 않아도

라는 의미를 이해못했습니다! 마우스를 어디에 올리는것인강쇼?

Reply:
userInfo?.profile 위에 마우스를 올리면 string | undefined라고 뜹니다! 논리 연산자나 널병합 연산자나 같은 결과를 만들어내지만 널 병합 연산자는 앞의 피연산자가 갖는 값이 한정되어 있습니다 null or undefined이죠! 하지만 논리 연산자는 앞의 피연산자가 null or undefined 말고도 false값, "", {}, []등 많은 값을 가져가서 결과를 만들어낼 수 있습니다. 현재 조건에 가장 최적화된 문법은 `??`가 아닐까하여 말씀드렸습니다!

Reply:
더 의미를 좁혀서 안정성을 가져간다는것이군요!! 좋습니다~! 확인입니다!

Reply:
[fix(client): || => ??](https://github.com/toppingskorea/toppings-client/pull/60/commits/551cd7f71214e8876c7e3d339c9d212b0739e35b)